hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/multitenant/OzoneTenant.java

```diff
@@ -24,16 +24,16 @@
 import org.apache.hadoop.ozone.om.multitenant.impl.SingleVolumeTenantNamespace;
 
 /**
- * Implements Tenant.
+ * Implements Tenant. TODO: Refactor/merge this with OmDBTenantInfo?
```

Comment:
ozoneTenant is in-memory version and OMDBTenantInfo is rocksDB version of the same entity. In-memory version maintains extra information.

Reply:
Revised class description.